couchdb/couchdb.go

```diff
@@ -430,6 +432,33 @@ func FindDocsRaw(db Database, doctype string, req interface{}, results interface
 	return json.Unmarshal(response.Docs, results)
 }
 
+// GetAllDocs returns all documents of a specified doctype. It filters
+// out the possible _design document.
+// TODO: pagination
+func GetAllDocs(db Database, doctype string, req *AllDocsRequest, results interface{}) error {
+	var response allDocsResponse
+	v, err := query.Values(req)
+	if err != nil {
+		return err
+	}
+	url := makeDBName(db, doctype) + "/_all_docs?included_docs=true&" + v.Encode()
```

Comment:
I think it will be cleaner to have v.Add("include_docs", true). 
**Note** s/included/include

Alternatively, we could also just POST the AllDocRequest, which we will need to do anyway for large number of keys.